scss/titles.scss

```diff
@@ -0,0 +1,18 @@
+.title-container {  
+    height: 90px;
+    width: 100px;
+
+    h2 {
+        left: 40px;
+        position: relative;
+        text-align: center;
+        top: 20px;   
+    }
+
+    &::before{
```

Comment:
Use something like these:
<img width="542" alt="captura de pantalla 2018-05-30 a la s 17 54 23" src="https://user-images.githubusercontent.com/36163179/40749587-825f6b50-6432-11e8-80d0-375d3859d2f5.png">

`I will let you to investigate top and left values`

Reply:
The final answer should look something like this 
`<h1 class="xxx-title">Front-End Tools</h1>`
`<h1 class="xxx-title">Back-End Tools</h1>`

Reply:
Remember that `position: absolute;` is 'relative' to the parent with `position: relative;`. That is why `position: absolute;` is used